packages/contracts/contracts/Poll.sol

```diff
@@ -181,6 +185,9 @@ contract Poll is Clone, Params, Utilities, SnarkCommon, IPoll {
     // store message batch size
     messageBatchSize = _messageBatchSize;
 
+    // store  state tree depth
+    stateTreeDepth = _stateTreeDepth;
```

Comment:
now that we can allow for a different state depth than the maci tree, we are going to need to ensure that when a user calls `joinPoll` the tree is not already full (before it would not be needed as the capacity check was done on the MACI contract already)